recipes-devtools/run-postinsts/run-postinsts_%.bbappend

```diff
@@ -29,3 +29,5 @@ do_install_ptest:append() {
 
 
 RDEPENDS:${PN}-ptest += "bash"
+PTESTS_FAST += "${PN}-ptest"
```

Comment:
Could you move this line up to line 12 - per [the styleguide](https://github.com/ni/meta-nilrt/blob/nilrt/master/kirkstone/docs/styleguide.md#layout)?